### PR TITLE
feat : created a Login component

### PR DESCRIPTION
**Pull request by GeetaPogul**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,81 @@
...
+
+      <Link to ="/Signup" > <i className="nav-icons fa fa-sign-in" aria-hidden="true"></i></Link>
+      <Link to="/Wishlist"> <i className="nav-icons far fa-heart"></i> 
+      </Link> 
+      <Link to="/CartPage" ><i className="nav-icons fas fa-shopping-cart"></i> </Link>
+
+
+
+
+     {/* notification badge  */}
```
Comment (Rohanmond) [inline src/App.js:47]: remove these comments

**Review by Rohanmond — APPROVED** [review]
code looks good to merge. Just remove unwanted comments

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,81 @@
...
+import "./components/HomePage/navigation.css"
+// import Navbar from "./components/HomePage/Navbar.js"
+
+// for navigation
+
+import {Routes, Route} from "react-router-dom"
+
+import CartPage from "./components/CartPage/CartPage";
+
+import Login from "./components/LoginPage/Login";
```
Comment (manish591) [inline src/App.js:12]: Try to use barrel imports for better readability

File: src/components/HomePage/Home.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+       </Routes>
+
+       </div>
+
+       {/* <Link to ="/Product"> Go to Products </Link>
+       <Routes>
+       <Route path='/Product' element={<Product/>} />
+
+       </Routes> */}
```
Comment (manish591) [inline src/components/HomePage/Home.js:26]: Remove comments if not required

File: src/components/HomePage/Home.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        <div className='cards-container'>
+
+
+        <div class="card">
+ 
+           < img class="card-image" src="../images/imac/imac1.jpg" alt="" />
+
+             {/* <div class="top-icon-red "> 
+                  <i class="fas fa-heart card-heart-icon" > </i> 
+              </div> */}
```
Comment (manish591) [inline src/components/HomePage/Home.js:41]: Remove this also if not required

File: src/components/LoginPage/Login.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+          <input
+            class="input-box"
+            type="email"
+            placeholder="johndoe@example.com"
+          />
+          <br />
+          <br />
+
+          <label class="input-label">
+            Password <span class="star"> * </span></label
```
Comment (manish591) [inline src/components/LoginPage/Login.js:32]: Use htmlFor attribute in the label and id in the label. It will increase the touch targets.

**Review by manish591 — APPROVED** [review]
Here are few Suggestions: 
* Keep consistent formatting, use prettier
* Remove commented code
* Also, some of your images are above 100kb, If possible try to compress them

rest all is good

**Pull request merged**